graphql_compiler/tests/test_compiler.py

```diff
@@ -4835,7 +4981,23 @@ def test_no_op_coercion_inside_fold(self):
                 )
             ])}
         '''
-        expected_sql = NotImplementedError
+        # TODO Not working b/c join descriptors for hyper edges not working properly
```

Comment:
What's the solution for this? Is there a plan for how to make hyper edges work?

Reply:
This isn't an issue with folds, it is a problem with what join descriptor is returned for a given edge. In particular, I think the from and to columns are reversed for this test case. I think @bojanserafimov  might know more about this